nvtabular/loader/tensorflow.py

```diff
@@ -210,6 +211,8 @@ def __init__(
         global_size=None,
         global_rank=None,
         drop_last=False,
+        sparse_list=None,
+        sparse_max=None,
```

Comment:
What do these two parameters do?  Can you add a description to the docstring?

Reply:
the parameters were updated in the framework specific dl doc strings.